Ginger/GingerCoreNET/Application Models/Execution/POM/POMExecutionUtils.cs

```diff
@@ -84,6 +84,20 @@ public ElementInfo GetCurrentPOMElementInfo()
             return selectedPOMElementInfo;
         }
 
+        public ElementInfo GetFriendlyElementInfo(Guid elementGuid)
+        {
+            Guid currentPOMElementInfoGUID = elementGuid;
```

Comment:
No need of currentPOMElementInfoGUID, directly use element guid

Reply:
Done